lib/roby/actions/models/action.rb

```diff
@@ -5,14 +5,21 @@ module Actions
         module Models
             # Basic description of an action
             class Action
+                Void = Object.new
```

Comment:
Just out of curiosity, why use this void object instead of nil for undefined example arguments?

Reply:
Because `nil` is a valid argument value.